src/styles/custom.css

```
@@ -41,9 +41,7 @@
     --sl-text-h4: var(--sl-text-lg) !important;
     --sl-text-h5: var(--sl-text-base) !important;*/
 
-    --sl-font-system:
-      ui-sans-serif, system-ui, Roboto, 'Atkinson Hyperlegible Next Variable', 'Inter Variable',
-      Arial, sans-serif;
+    --sl-font-system: ui-sans-serif, system-ui, Roboto, 'Inter Variable', Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)